src/template.c

```diff
@@ -8108,7 +8108,7 @@ void TemplateMixin::semantic(Scope *sc)
     printf("\tdo semantic() on template instance members '%s'\n", toChars());
 #endif
     Scope *sc2 = argscope->push(this);
-    size_t deferred_dim = Module::deferred.dim;
+    //size_t deferred_dim = Module::deferred.dim;
```

Comment:
It's kept for documentation purposes - see what is written 10 lines or so down.  (https://github.com/D-Programming-Language/dmd/commit/266b842acba9da7e63c8c59ef798cc4d011caf94)